README.md

```diff
@@ -161,6 +161,7 @@ $ curl -H "Authorization: Basic <base64_encoded_token>" http://localhost:8083/v1
 | **aggregation-cycle-time** | string | no | `15s` | Time (in seconds) to wait between each aggregation cycle. |
 | **debug** | bool | no | false | Enable debug logging. |
 | **detector-port** | string | no | `:8083` | Detector HTTP server port |
+| **detector-datacenter** | []string | no | N/A | List of datacenters where detector is running. If no datacenters are provided, aggregator will only reach out to nodes in `$NOMAD_DC` datacenter. |
```

Comment:
What does it look like to specify multiple here? Do you specify some comma or otherwise separated list or specify the option twice? Wondering if it should be plural, for example.

Do we have example invocations to show this?

Reply:
Oh, should we add that NOMAD_DC is always included even if not listed in the list?

Reply:
I have intentionally kept it as singular and not plural. You specify this option multiple times e.g. `--dc A --dc B --dc C`. I am using the short flag, but one can also specify using `--detector-datacenter`. This is similar to `enforce-health-check` (Singular) which is a slice of strings where the operator can enforce multiple health checks. 

If the operator doesn't set `--detector-datacenter` list when running `aggregator`, `aggregator` will just use `$NOMAD_DC` datacenter.